google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java

```diff
@@ -206,10 +206,15 @@ Logging getLogging() {
   }
 
   private LogEntry logEntryFor(ILoggingEvent e) {
-    String payload = e.getFormattedMessage();
+    StringBuilder payload = new StringBuilder(e.getFormattedMessage());
+    for (StackTraceElement stackEl : e.getCallerData()) {
```

Comment:
`element` or `stackElement` would be more idiomatic for the variable name

Reply:
Done